### PR TITLE
Add option for log suppression

### PR DESCRIPTION
**Pull request by mrginglymus**
I think this is all the necessary boilerplate required? Will update docs as well, ofc.

### CODE REVIEW

**Review by oleg-nenashev — APPROVED** [review]
yes, it would be a good addition

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
     <useBeta>true</useBeta>
 
     <!-- Jenkins Plug-in Dependencies Versions -->
-    <checks-api.version>1.5.0</checks-api.version>
+    <checks-api.version>1.6.0-rc209.3e34bea21942</checks-api.version>
```
Comment (jglick) [inline pom.xml:28]:
```suggestion
    <checks-api.version>1.6.0-rc209.3e34bea21942</checks-api.version> <!-- TODO https://github.com/jenkinsci/checks-api-plugin/pull/93 -->
```

File: src/main/resources/lib/github-checks/status/properties.jelly [hunk 2]
```diff
@@ -13,4 +13,8 @@
     <f:checkbox/>
   </f:entry>
 
+  <f:entry title="Suppress log output in checks" field="suppressLogs">
```
Comment (jglick) [inline src/main/resources/lib/github-checks/status/properties.jelly:16]: consider adding `help-suppressLogs.html` to this folder

File: pom.xml [hunk 3]
```diff
@@ -25,7 +25,7 @@
     <useBeta>true</useBeta>
 
     <!-- Jenkins Plug-in Dependencies Versions -->
-    <checks-api.version>1.5.0</checks-api.version>
+    <checks-api.version>1.6.0-rc209.3e34bea21942</checks-api.version> <!-- TODO https://github.com/jenkinsci/checks-api-plugin/pull/93 -->
```
Comment (XiongKezhi) [inline pom.xml:28]: I'll release it once I figure out the build problem: https://github.com/jenkinsci/checks-api-plugin/pull/93#issuecomment-791898907

**Review by timja — COMMENTED** [review]
any other docs updates needed in readme or elsewhere?

**Comment by XiongKezhi** [comment]
> any other docs updates needed in readme or elsewhere?

I'll update the readme with a new screenshot and explains the status checks properties.

**Pull request merged**